php-nginx/php.ini

```diff
@@ -586,7 +586,7 @@ html_errors = Off
 ; Example:
 ;error_log = php_errors.log
 ; Log errors to syslog (Event Log on Windows).
-error_log = /var/log/app_engine/php-error.log
+error_log = /var/log/app_engine/application.log
```

Comment:
This allows the fluentd logger to detect the log file. Currently, it is not capturing the log output because it doesn't match the [fluentd configuration](https://github.com/GoogleCloudPlatform/appengine-sidecars-docker/blob/master/fluentd_logger/managed_vms.conf#L4)